app/components/TopicPicker/index.js

```diff
@@ -23,36 +18,78 @@ const TopicPicker = React.createClass({
 
   propTypes: {
     topics: React.PropTypes.array.isRequired,
+    onActivateTopic: React.PropTypes.func,
+    currentTopic: React.PropTypes.string.isRequired,
+    //colors: React.PropTypes.array.isRequired
   },
 
-  activateTopic(topicId, props) {
-    console.log(topicId);
+  componentDidMount() {
+      this.activateTopic('1');
```

Comment:
this should not be hardcoded. also why are we doing topic stuff in the highlights PR?